### PR TITLE
Bump to Django 2.2.13

### LINKED ISSUE #109: Upgrade Django to 2.2.13
> Django versions 3.0.7 and 2.2.13 will be released on Wednesday, June 3, 2020 around 1000 UTC. They will fix two security defects with severity of "medium" and "low". 

See:

- https://travis-ci.org/github/azavea/docker-django/builds/672582070

### PR DESCRIPTION
**Pull request by rbreslow**
Bump Django to accommodate security release.

See: https://www.djangoproject.com/weblog/2020/jun/03/security-releases/

Resolves #109 

---

**Testing**

See Travis CI build: https://travis-ci.org/github/azavea/docker-django/builds/694310167.

### CODE REVIEW

File: 2.2/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 gunicorn==20.0.4
-django==2.2.12
+django==2.2.13
 psycopg2==2.8.5
 gevent==1.4.0
```
Comment (coalkettler) [inline 2.2/requirements.txt:4]: Apparently `gevent==1.5.0` came out a few days after our last bump and should work with all the versions of Python we support. Could we bump that while we're at it?

**Review by coalkettler — APPROVED** [review]
Looks good! I called attention to another dependency we could bump but we don't necessarily have to do that as part of this PR.

File: 2.2/requirements.txt (hunk 1 above)
Comment (rbreslow) [reply 2.2/requirements.txt:4]: I'm aware that `gevent==1.5.0` came out. It broke the builds with what looks like a dependency on a new native library. I tried to satisfy it, but wasn't able to make much progress, and decided to just skip the bump.

I'll create an issue to give it more scrutiny.

Comment (coalkettler) [reply 2.2/requirements.txt:4]: Ooh, nice catch, agreed on creating an issue. Thanks!

**Pull request merged**